iot-hub/Samples/service/IotHubSasCredentialAuthenticationSample/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Azure;
+using Azure.Core;
+using CommandLine;
+using Microsoft.Azure.Devices;
+using System;
+using System.Globalization;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IotHubSasCredentialAuthenticationSample
```

Comment:
I think this name was a relic from back when we planned on extending AzureSasCredential with a new class IotHubSasCredential. Since we aren't doing that anymore, shouldn't this sample be renamed to match?